test/chairs.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python3
+"""
+Test chars and chair-mp mapping metadata
+"""
+import pandas as pd
+import unittest
+import warnings
+
+
+
+
+class ChairHog(Warning):
+
+    def __init__(self, m):
+        self.message = f"There following MPs sit in two chairs in {m}:."
+
+
+    def __str__(self):
+        return self.message
+
+
+
+
+class ChairInWrongTid(Warning):
```

Comment:
I dont understand this one?

Reply:
The chair is in the wrong time period -- e.g. an enkammar chair appearing in the two chamber period or vice versa.

Reply:
Maybe call it: ChairInWrongTimePeriod(Warning)?

Yes. I think it is good to change the class name. If I don't understand this now. It will be harder to understand this in 6 mo or 3 y.

Reply:
These warning classes are just glorified print statements in this use case, but OK, will do.

Reply:
Sure. I also wanted to change it in other places where we use the OOR. The same with WrongTid.

It's not much work now and it will save us further down the line. It is also a good coding habit. There is the saying that you write code for humans, not computers.

Reply:
a8cc60a2d0c6595c4b0130426c5e1e7b6888282c